### ISSUE TITLE
Tags is not classified by type like the old report

### ISSUE THREAD

**Issue by otoniel-isidoro-sofist** [serenity-bdd/serenity-core#1421]
The new report shows tags a little disorganized (not ordered and not classified by type) as is showed on the example below:
![image](https://user-images.githubusercontent.com/39963236/47274091-9952d580-d576-11e8-9d13-b585a892d0b3.png)

The old report organizes the tags by type as shown below:
![image](https://user-images.githubusercontent.com/39963236/47274094-a66fc480-d576-11e8-8362-a96e103300b4.png)

I've tried to improve it on the home.ftl template but nothing worked as I expected. Below is my attempt code on the home.ftl (I'm not sure if the code syntax is right):
```
<#if tagResults?has_content >
<div class="row">
    <div class="col-sm-12">
        <h3>Tags</h3>
        <#list tagTypes as tagType >
            <br/><h2>${tagType}:</h2><br/>
            <#list tagResults as tagResult >
                <#if tagResult.tag.type == tagType >
                    <span>
                        <a href="${tagResult.report}">
                            <span class="badge"
                                  style="background-color:${tagResult.color}; margin:1em;padding:4px;"><i
                                    class="fa fa-tag"></i> ${tagResult.label}&nbsp;&nbsp;&nbsp;${tagResult.count}</span>
                        </a>
                    </span>
                </#if>
            </#list>
        </#list>

    </div>
</div>
</#if>
```
Any tip how can I make it shows the tags by types?

**Comment by nbarrett** [comment]
Hi @otoniel-isidoro-sofist - try with [2.0.10](https://github.com/serenity-bdd/serenity-core/releases/tag/v2.0.10) - this release now separates by tag type.

**Comment by wakaleo** [comment]
Implemented in 2.0.10 and 2.0.11
